frontend/dockerfile/dockerfile2llb/convert.go

```diff
@@ -1069,6 +1071,22 @@ func dispatchCopy(d *dispatchState, cfg copyConfig) error {
 		}
 	}
 
+	isHTTPSource := func(src string) bool {
```

Comment:
nit: could we avoid closure for this and just define helper function

Reply:
fixed